iOS/Projects/Features/Record/Sources/WorkoutSelectScene/View/WorkoutPeerSelectViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  WorkoutPeerSelectViewController.swift
+//  RecordFeature
+//
+//  Created by MaraMincho on 11/19/23.
+//  Copyright © 2023 kr.codesquad.boostcamp8. All rights reserved.
+//
+
+import DesignSystem
+import UIKit
+
+// MARK: - WorkoutPeerSelectViewController
+
+final class WorkoutPeerSelectViewController: UIViewController {
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    view.backgroundColor = DesignSystemColor.primaryBackground
+
+    setup()
+  }
+
+  var dataSource: UICollectionViewDiffableDataSource<Int, UUID>!
```

Comment:
P2

여기도 강제언래핑으로 보여지는데요!
사용하신걸 확인해보니 아래처럼 선언해도 괜찮을것 같아요!

```suggestion
  var dataSource: UICollectionViewDiffableDataSource<Int, UUID>?
```

Reply:
P3

또한 셀이 2개밖에 없어서 DiffableDatasource를 사용하지 않아도 될것 같다는 의견입니다!

스프린트할 때, 리뷰어님들께서 재사용되는 셀이 없을때는 CollectionView나 TableView를 사용하지 않는게 좋을것 같다고 하셨던게 기억나네요.
오버엔지니어링처럼 보여서 그러신걸까요 ㅎㅎ.. 이유는 찾지 못한것같습니다

Reply:
사실 CollectionView로 구현한 이유는, 다양한 환경을 고려해서 였습니다. 현재는 랜덤 매칭과 혼자 달리기 뿐이지만, 다양한 조건을 추가할 수 있다고 생각했습니다. 가령 클럽 멤버와 달리기나 친구와 달리기 그리고 동시에 시간에 맞춰 달리기 등 다양한 메뉴가 추가될 수 있다고 생각했습니다. 유연한 구조를 생각하다 보니, 재사용 관점에서의 diffable datasource는 그닥 구미가 당기지 않는 선택지 같네요...

Reply:
아닙니다!
재사용관점에서 Diffable Datasource는 제 생각에 가장 좋은 선택지 같아요!
제가 말씀드린건 그저 셀이 2개인데 CollectionView를 사용하는게 오버엔지니어링이 될 수 있겠다고 말씀드린건데요!
다함님께서 납득할만한 이유를 설명해주셔서 변경안하셔도 될것같아요 🤗